<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pienoisjunarata</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>web.png</normaloff>web.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGraphicsView" name="graphicsView"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Pituus</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="SliderPituus">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>300</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdNumber"/>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Kulma</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="SliderKulma">
              <property name="minimum">
               <number>-628</number>
              </property>
              <property name="maximum">
               <number>628</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdNumber_2"/>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="BtnPuu">
            <property name="text">
             <string>Puu</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnMutka">
            <property name="text">
             <string>Mutka</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QPushButton" name="BtnSuora">
            <property name="text">
             <string>Suora</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnVaihde">
            <property name="text">
             <string>Vaihde</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="BtnStopperi">
            <property name="text">
             <string>Stopperi</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnNelio">
            <property name="text">
             <string>Nelio</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
